src/xcode/ENA/ENA/Source/Views/Home Screen/Cells/HomeTestResultCell.swift

```diff
@@ -31,99 +31,9 @@ class HomeTestResultCell: HomeCardCollectionViewCell {
 	@IBOutlet weak var image: UIImageView!
 	@IBOutlet weak var body: UILabel!
 	@IBOutlet weak var button: ENAButton!
-
-	weak var delegate: TestResultCollectionViewCellDelegate?
+	weak var delegate: HomeTestResultCellDelegate?
 
 	@IBAction func buttonTapped(_: UIButton) {
```

Comment:
I think, it's the same cell as `SubmitCollectionViewCell`, so please remove duplication.

Reply:
I added a `HomeCardCellButtonDelegate` that replaced the duplicated delegates